lib/LedgerSMB/Report.pm

```diff
@@ -281,8 +281,8 @@ sub _render {
                        $srt_b = $srt_b->to_sort
                            if eval { $srt_b->can('to_sort') };
                    }
-                   no warnings 'numeric';
-                   $srt_a <=> $srt_b or $srt_a cmp $srt_b;
+
+                   $srt_a cmp $srt_b;
```

Comment:
I agree with @einhverfr here.
Also, however we resolve it, some comments explaining what, why and how the code works/is-the-way-it-is could help prevent others from looking to simplify without deeper knowledge